frontend/public/locales/en/quiz.json

```diff
@@ -91,8 +91,8 @@
     },
     "question-where": {
       "option-canada-ft": "In Canada",
-      "option-canada-pt-60-or-more": "In Canada <span>more</span> than 6 months a year",
-      "option-canada-pt-less-than-60": "In Canada <span>less</span> than 6 months a year",
+      "option-canada-pt-60-or-more": "In Canada more than 6 months a year",
+      "option-canada-pt-less-than-60": "In Canada less than 6 months a year",
```

Comment:
If you remove the `<span>` here, then you also have to modify the associated lines in `src/components/quiz/questions/QuestionWhere.tsx`.

If you're not comfortable doing that, I would suggest to leave the `<span>` in and ask one of the developers to remove it.